exporter/opentelemetry-exporter-prometheus-remote-write/setup.cfg

```diff
@@ -0,0 +1,49 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
I think we've moved to pyproject.toml e.g. https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/exporter/opentelemetry-exporter-richconsole/pyproject.toml

@open-telemetry/opentelemetry-python-contrib-maintainers is there an easy way to migrate this?